.eslintrc

```diff
@@ -12,18 +12,7 @@
   "overrides": [
     {
       "files": ["test/**"],
-      "plugins": ["jest"],
-      "rules": {
-        "global-require": "off",
-        "no-await-in-loop": "off",
-        "no-console": "off",
-        "import/no-dynamic-require": "off",
-        "jest/no-disabled-tests": "warn",
-        "jest/no-focused-tests": "error",
-        "jest/no-identical-title": "error",
-        "jest/prefer-to-have-length": "warn",
-        "jest/valid-expect": "error"
-      }
+      "plugins": ["jest", "eslint-config-4catalyzer-jest"]
```

Comment:
wrong place, should go in the top level `extends`, see my changes in cli

Reply:
actually I have a question about [it](https://github.com/4Catalyzer/cli/pull/19#discussion_r249580091)